inc/settings.php

```diff
@@ -24,47 +24,257 @@ function siteorigin_corp_settings_init() {
 
 	SiteOrigin_Settings::single()->configure( apply_filters( 'siteorigin_corp_settings_array', array(
 
-		// Branding.
-		'branding' => array(
-			'title' => esc_html__( 'Branding', 'siteorigin-corp' ),
+		// Header.
```

Comment:
I've never seen this before and this is very minor but, this feels a tad redundant as the settings group is named in the array itself. Not a big deal, just an observation.